### PR TITLE
Introduce a new tool/function to visualize a pb model as requested by #8854

### PR DESCRIPTION
**Pull request by jubjamie**
Issue #8854 asked for a tool to easily visualize .pb models as graphs. I pointed out it was possible to do with Tensorboard so i've created a function to allow someone to pass a .pb model and a tensorboard log directory to then view the imported .pb graph.
I guess this is a temporary measure until a decision is made on creating something a bit more friendly.

Also, For some reason I think that this PR is including some previously merged commits and I don't know how to get rid of them on github! I thought i'd sync'd everything but i'm a bit rubbish at this. I guess those can be safely ignored. Thanks

### CODE REVIEW

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import tensorflow as tf
+from tensorflow.python.platform import gfile
+from tensorflow.python.framework.ops import reset_default_graph
+
+def import_to_tensorboard(modeldir,logdir):
```
Comment (drpngx) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: Could you make that a docstring?

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# import_to_tensorboard(modeldir,logdir) provides a quick way to visualize 
+# a .pb model in Tensorboard.
+# Created in response to issue #8854
```
Comment (drpngx) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: remove this line

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#	logdir: The location for the Tensorboard log to begin visualisation from.
+# Usage:
+# 	Call this function with your model location and desired log directory.
+#	Launch Tensorboard by pointing it to the log directory.
+#	View your imported .pb model as a graph.
+#
+#
+  reset_default_graph()
+  with tf.Session() as sess:
+    model_filename =modeldir
```
Comment (drpngx) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: add space after `=`.

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    with gfile.FastGFile(model_filename, 'rb') as f:
+      graph_def = tf.GraphDef()
+      graph_def.ParseFromString(f.read())
+      g_in = tf.import_graph_def(graph_def)
+
+  pbVisualWriter = tf.summary.FileWriter(logdir)
```
Comment (drpngx) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: our style guide uses `pb_visual_writer`

File: tensorflow/python/tools/import_pb_to_tensorboard.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import tensorflow as tf
+from tensorflow.python.platform import gfile
+from tensorflow.python.framework.ops import reset_default_graph
+
+def import_to_tensorboard(modeldir,logdir):
+# import_to_tensorboard(modeldir,logdir) provides a quick way to visualize 
+# a .pb model in Tensorboard.
+# Created in response to issue #8854
+# Args:
+# 	modeldir: The location of the .pb model to visualize
```
Comment (drpngx) [inline tensorflow/python/tools/import_pb_to_tensorboard.py]: please explain at least one pb = protobuf

**Review by drpngx — CHANGES_REQUESTED** [review]

**Comment by jubjamie** [comment]
@drpngx changes have been made. I'm unsure as to why my previous commits from a separate PR have ended up in here. Sorry for the delay in getting this out.

**Comment by dandelionmane** [comment]
The change looks good to me. And, please make the comment into a docstring per @drpngx's request.

**Comment by jubjamie** [comment]
Sorry what so you mean by docstring? Can you link an example?

**Review by teamdandelion — DISMISSED** [review]
Looks good, please just make the comment into a docstring (https://www.python.org/dev/peps/pep-0257/)

**Comment by vrv** [comment]
@dandelionmane @drpngx I made the changes for the user, PTAL

**Review by drpngx — APPROVED** [review]

**Comment by drpngx** [comment]
Jenkins, test this please.

**Pull request merged**